components/Users/WrapperUser.jsx

```diff
@@ -6,10 +6,13 @@ import { Box, Paper, Typography } from "@mui/material";
 import FoodAddForm from "./FoodAddForm";
 import { FoodEntriesPerEmail } from "./FoodEntriesPerEmail";
 import { MealEntriesPerEmail } from "./MealEntriesPerEmail";
+import FoodEntryList from "./FoodEntryList";
+import { motion } from "framer-motion";
 
 const UserEntry = ({ sessionUser }) => {
   const sessionEmail = sessionUser?.email;
```

Comment:
just `email` will be enough here.
`const email = sessionUser?.email;` or `const {email} = sessionUser;`